data.sql

```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Agumon', 2020-02-03, 0, 1, 10.23);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', 2018-11-15, 0, 2, 8);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Pikachu', 2021-01-07, 0, 1, 15.04);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', 2017-05-12, 0, 5, 11);
```

Comment:
- [ ] You are assigning 0 for `neutered` to all animals, please keep in mind that all animals are neutered except `Pikachu`. Please fix this issue and make sure you assign 0 only to `Pikachu` and 1 for all other animals.